guides/common/modules/proc_preparing-a-host-for-external-databases.adoc

```diff
@@ -7,11 +7,11 @@ Subscriptions for Red{nbsp}Hat Software Collections and {RHEL} do not provide th
 
 .Prerequisites
 
-* The {RHEL} 7 server must meet {Project}'s link:{BaseURL}installing_satellite_server_from_a_connected_network/preparing-environment-for-satellite-installation#satellite-storage-requirements_satellite[Storage Requirements].
+* The {RHEL} 7 server must meet {Project}'s link:{InstallingProjectDocURL}satellite-storage-requirements_{project-context}[Storage Requirements].
 
 .Procedure
 
-. Use the instructions in link:{BaseURL}installing_satellite_server_from_a_connected_network/installing-satellite-server-connected#attaching-satellite-infrastructure-subscription_satellite[Attaching the {Project} Infrastructure Subscription] to attach a {Project} subscription to your server.
+. Use the instructions in link:{InstallingProjectDocURL}attaching-satellite-infrastructure-subscription_{project-context}[Attaching the {Project} Infrastructure Subscription] to attach a {Project} subscription to your server.
```

Comment:
They are completely redundant from the technical side of things. Tahlia told me that she was using them because otherwise, links starting with attributes won't highlight as links in her text editor, it was Atom if I remember correctly. In this PR I tend to use the less manual method of manipulating links so I did not touch the `link` part.
This is something that can be safely done by running sed to substitute `s/link://g' and then reviewing if nothing breaks. How about opening an issue for this and doing this in a separate PR?